scripts/move_tags_to_custom_fields.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import csv
+from closeio_api import Client as CloseIO_API
+
+parser = argparse.ArgumentParser(description='uploads custom fields from csv')
+parser.add_argument('csvfile', type=argparse.FileType('r'), help='csv file')
+parser.add_argument('--api_key', '-k', required=True, help='API Key')
+parser.add_argument('--development', '-d', action='store_true', help='Use a development (testing) server rather than production.')
+
+args = parser.parse_args()
+
+tag_templates = {}
+
+c = csv.DictReader(args.csvfile, delimiter=',', quotechar='"', fieldnames=['tag', 'custom_field_name', 'custom_field_value'])
+c.next()
+for r in c:
+    if r:
+        assert len(r) == 3, 'Invalid csv format'
+        tag_templates[r['tag']] = (r['custom_field_name'], r['custom_field_value'])
+
+
+api = CloseIO_API(args.api_key, development=args.development)
+has_more = True
+offset = 0
+
+while has_more:
+    resp = api.get('lead', data={
+        'query': '*',
```

Comment:
just fyi, since our default lead api sorting is by date edited, I'm going to change the default query here from `*` to be `sort:created` so that the ordering is consistent upon modifying the list of leads so that pagination truly gets all records